splink/estimate_u.py

```diff
@@ -51,6 +52,12 @@ def _proportion_sample_size_link_only(
     return proportion, sample_size
 
 
+def _get_duckdb_salting(max_pairs):
+    logged = math.log(max_pairs, 10)
```

Comment:
this is a heuristic. 

```
max_pairs, salting
1e+00 1
1e+01 1
1e+02 1
1e+03 1
1e+04 1
1e+05 3
1e+06 7
1e+07 16
1e+08 40
2e+08 52
1e+09 98
1e+10 245
1e+11 611
```

Reply:
Might be useful to have a very brief explanation as a comment, as otherwise this function is maybe a little cryptic